client/pages/Home/Components/BookRows.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import BookDetails from '@components/BookDetails';
+
+const BookRows = () => {
```

Comment:
I'm not sure we need this component. What do you think ?

Reply:
I think it helps with simplicity and easy alignment of the BookDetails component. I had tried not using it (I really didn't like the result). I am open to any suggestion you have regarding this... I would be making a PR effecting all the changes that have been resolved now

Reply:
What I mean is, is it necessary for this to be a completely separate component? This row can be in the home component instead of being its own separate component.

What I expected you to say was that this component will be re-used on every page where we want to display a row of books. 

Please rename it to `BookRow`

Reply:
Alright, Thank you. I would do that right away and move it the component into BookDetails...would that be okay?